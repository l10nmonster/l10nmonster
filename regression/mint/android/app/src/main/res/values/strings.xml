<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="base">Base: <strike>%1$s</strike></string>
    <string name="winter"><![CDATA[Winter is <b>coming</b>]]></string>
    <string name="frozen">Frozen: <strike>%1$s</strike></string>
    <string name="house">house: %1$s motto: %2$s</string>
    <string name="nothing"><![CDATA[I know 100%%]]></string>
    <string name="everything"><![CDATA[I know<strong><font color="%1$s">%2$s</font>%%</strong>]]></string>
</resources>
