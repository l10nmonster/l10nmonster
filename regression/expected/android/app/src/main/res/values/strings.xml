<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="winter"><![CDATA[Winter is <b>coming</b>]]></string>
    <string name="house">house: %1$s motto: %2$s</string>
</resources>
